### PR TITLE
[stable/rabbitmq] Rename *_PORT env vars to *_PORT_NUMBER

### PR DESCRIPTION
**Pull request by tompizmor**
In order to avoid environment variable collision with k8s services, we can add a suffix to the *_PORT environment variables so we can be sure that any service name will work.

### CODE REVIEW

File: stable/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 0.5.2
```
Comment (sameersbn) [inline stable/rabbitmq/Chart.yaml]: @tompizmor since this is a breaking change, I think we should bump the minor version of the chart.

**Comment by sameersbn** [comment]
@k8s-bot ok to test

File: stable/rabbitmq/Chart.yaml (hunk 1 above)
Comment (tompizmor) [reply stable/rabbitmq/Chart.yaml]: It would be a breaking change if someone just update the docker image in the chart but not if they run `helm upgrade`

**Comment by unguiculus** [comment]
Marking as stale. Please update within the next week.

**Comment by tompizmor** [comment]
PR updated @unguiculus

**Comment by unguiculus** [comment]
Please bump chart version.

ping @sameersbn

**Comment by sameersbn** [comment]
lgtm!

**Pull request merged**